ci/build.py

```diff
@@ -130,6 +147,14 @@ def script_name() -> str:
                         help="Build the container",
                         action='store_true')
 
+    parser.add_argument("--build_only",
```

Comment:
Would you elaborate in what use case you would like to build the containers only? As discussed offline, it would be good if the containers get built automatically - I can't imagine any use case in which you would like to run with an outdated docker image.